README.md

```diff
@@ -276,6 +276,22 @@ To clear helm release, do the following:
   }
   ```
 
+#### Clear Helm Release on Failed Build
```

Comment:
is there a reason we wouldn't want to do this by default given we're capturing the pods logs? I guess if you wanted to check the pod environment variables etc? but we should capture the pod manifest as well maybe ?

Reply:
The reason is simply that I don't presume to know how other teams like to work.

For now it's just safer to add the feature as an "opt-in". Then platops can do further considerations as to the opportunity to enable this by default (which I would imagine would be the best thing to do) and what other changes are needed to support that properly.